scrapy/czlscrape/__init__.py

```diff
@@ -1,7 +1,11 @@
 import os
+import time
+import logging
 
 if 'SENTRY_DSN' in os.environ:
     import logging
     from raven.handlers.logging import SentryHandler
     from raven.conf import setup_logging
     setup_logging(SentryHandler(os.environ['SENTRY_DSN'], level=logging.WARN))
+
+logging.Formatter.converter = time.gmtime
```

Comment:
What I don't like about local datetimes is the daylight saving differences, particularly having to cope with them. Also, events that come from any other source (think AWS services or infrastructure) will be in UTC. It's easier to correlate events when they are expressed in the same timezone.